### PR TITLE
LTP: emulate network.sh script for networking tests

### PR DESCRIPTION
**Pull request by pevik**
This is needed as /opt/ltp/testscripts/network.sh loads test_net.sh,
which sets various environment variables needed by networking tests.

@richiejp now we might reconsider stopping wickedd & NetworkManager only for networking tests.

### CODE REVIEW

**Comment by okurz** [comment]
LGTM

File: tests/kernel/run_ltp.pm [hunk 1]
```diff
@@ -246,6 +246,7 @@ sub run {
     my $cmd_exclude = get_var('LTP_COMMAND_EXCLUDE') || '$^';
     my $timeout     = get_var('LTP_TIMEOUT')         || 900;
     my $is_posix    = $cmd_file =~ m/^\s*openposix\s*$/i;
+    my $is_network  = $cmd_file =~ m/^\s*net\./;
```
Comment (richiejp) [inline tests/kernel/run_ltp.pm]: This may need to include `rpc_tests` and `tirpc_tests` as well.

**Comment by richiejp** [comment]
Yeah, as there is now an `if` statement anyway, you might as well only do it for network tests.

File: tests/kernel/run_ltp.pm (hunk 1 above)
Comment (pevik) [inline tests/kernel/run_ltp.pm]: @richiejp I'm still waiting for upstream reply about it, I'll test that. But if they need these variables, than it's an upstream bug (it should be added into /opt/ltp/testscripts/network.sh and have prefix 'net.').

**Comment by pevik** [comment]
Stop networking daemons only for networking tests. I suppose it's better to keep two related changes in one commit (splitting these into two doesn't make much sense to me).

**Comment by richiejp** [comment]
Looks fine to me. I am not sure about the inline comments, because we have the git blame feature amongst other reasons, but I'd rather leave that debate for another time. I will probably merge after lunch.

**Pull request merged**